doc/source/installation.rst

```diff
@@ -139,6 +139,8 @@ via pip.
 
   pip install ray[dashboard]
 
+If you are using a command shell like ``zsh``, you may need to put put quotes around ``ray[dashboard]``.
```

Comment:
why not just `pip install "ray[dashboard]"` in L140?

Reply:
Actually, can we get rid of `ray[dashboard]` now? @ijrsvt @simon-mo

Reply:
Yeah `ray[dashboard]` is an empty target now for compatibility reason.